medcat/config_meta_cat.py

```diff
@@ -5,6 +5,7 @@
 class General(MixingConfig, BaseModel):
     """The General part of the MetaCAT config"""
     device: str = 'cpu'
+    """Device to used by the module"""
     disable_component_lock: bool = False
```

Comment:
what does this do?

Reply:
I briefly looked at this a week or two ago, and what I wrote down as a doc string was:
```
    """Whether to use the MetaCAT component lock.

    If set to False (the default), a component lock is used that forces usage
    only on one thread at a time.
    If set to True, the component lock is not used."""
 ```

Reply:
I'll add this to the config description